src/screens/Library/components/InstallModal/index.tsx

```diff
@@ -24,10 +38,14 @@ import {
   Path,
   Runner
 } from 'src/types'
+import Dialog from '../../../../components/UI/Dialog'
```

Comment:
We have relative imports configured on Heroic so you can do it like:
`import Dialog from 'src/components/UI/Dialog'` and so on.